AmplifyPlugins/Predictions/AWSPredictionsPlugin/Support/Internal/ErrorHandling/Textract+PredictionsErrorConvertible.swift

```diff
@@ -0,0 +1,130 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import AWSTextract
+import Amplify
+import ClientRuntime
+
+
+extension AWSTextract.HumanLoopQuotaExceededException: PredictionsErrorConvertible {
+    var predictionsError: PredictionsError {
+        .service(
+            .init(
+                description: "",
```

Comment:
same question as previous comment, is the empty strings intended?

Reply:
Addressed in [f13a59a](https://github.com/aws-amplify/amplify-swift/pull/3257/commits/f13a59a4dd5e7767cc614347a072aaa41ec81089)